openreview/conference/templates/areachairWebfield.js

```diff
@@ -413,29 +410,39 @@ var renderStatusTable = function(profiles, notes, completedReviews, metaReviews,
     var messageCount = localStorage.getItem('messageCount');
     if (!reviewerMessages || !messageCount) {
       $('#message-reviewers-modal').modal('hide');
-      promptError('Could not send reminder emails at this time. Please refresh the page and try again.');
+      promptError('Could not send emails at this time. Please refresh the page and try again.');
     }
     JSON.parse(reviewerMessages).forEach(postReviewerEmails);
 
     localStorage.removeItem('reviewerMessages');
     localStorage.removeItem('messageCount');
 
     $('#message-reviewers-modal').modal('hide');
-    promptMessage('Successfully sent ' + messageCount + ' reminder emails');
+    promptMessage('Successfully sent ' + messageCount + ' emails');
   };
 
   var sortOptionHtml = Object.keys(sortOptions).map(function(option) {
     return '<option value="' + option + '">' + option.replace(/_/g, ' ') + '</option>';
   }).join('\n');
 
   var sortBarHtml = '<form class="form-inline search-form clearfix" role="search">' +
-    '<strong>Sort By:</strong> ' +
-    '<select id="form-sort" class="form-control">' + sortOptionHtml + '</select>' +
-    '<button id="form-order" class="btn btn-icon"><span class="glyphicon glyphicon-sort"></span></button>' +
+    '<div id="div-msg-reviewers" class="btn-group" role="group">' +
+      '<button id="message-reviewers-btn" type="button" class="btn btn-icon dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="Select papers to message corresponding reviewers" disabled="disabled">' +
+        '<span class="glyphicon glyphicon-envelope"></span> &nbsp;Message Reviewers ' +
+        '<span class="caret"></span>' +
+      '</button>' +
+      '<ul class="dropdown-menu" aria-labelledby="grp-msg-reviewers-btn">' +
+        '<li><a id="msg-all-reviewers">All Reviewers of selected papers</a></li>' +
+        '<li><a id="msg-submitted-reviewers">Reviewers with submitted reviews for selected papers</a></li>' +
```

Comment:
Small nitpick, but could you change these 2 options to "Reviewers of selected papers with submitted reviews", and "Reviewers of selected papers with unsubmitted reviews".